### ISSUE TITLE
Resource '/app/index.htm' failed unexpectedly

### ISSUE THREAD

**Issue by mariussoutier** [ldaniels528/trifecta#20]
In 0.19.0, I get the following error:

`Resource '/app/index.htm' failed unexpectedly`

~~~
4216 [main] INFO  com.github.ldaniels528.trifecta.TrifectaShell$ - Open your browser and navigate to http://0.0.0.0:8888
12791 [EmbeddedWebServer-akka.actor.default-dispatcher-4] ERROR com.github.ldaniels528.trifecta.rest.WebContentActor - Resource '/app/index.htm' failed unexpectedly WARNING arguments left: 1
13054 [EmbeddedWebServer-akka.actor.default-dispatcher-4] ERROR com.github.ldaniels528.trifecta.rest.WebContentActor - Resource '/app/favicon.ico' failed unexpectedly WARNING arguments left: 1
23713 [main-SendThread(prd-zookeeper00-03.gut.dglecom.net:2181)] WARN  org.apache.zookeeper.ClientCnxn - Client session timed out, have not heard from server in 20045ms for sessionid 0x0
31225 [EmbeddedWebServer-akka.actor.default-dispatcher-4] ERROR com.github.ldaniels528.trifecta.rest.WebContentActor - Resource '/app/index.htm' failed unexpectedly WARNING arguments left: 1
31472 [EmbeddedWebServer-akka.actor.default-dispatcher-3] ERROR com.github.ldaniels528.trifecta.rest.WebContentActor - Resource '/app/favicon.ico' failed unexpectedly WARNING arguments left: 1
43852 [main-SendThread(prd-zookeeper00-01.gut.dglecom.net:2181)] WARN  org.apache.zookeeper.ClientCnxn - Client session timed out, have not heard from server in 20032ms for sessionid 0x0
~~~

**Comment by ldaniels528** [comment]
I'll take a look; however, I'd encourage you to use the new Trifecta UI Play application. To build it, do the following:

sbt dist

Then unzip the generated zip file, and run the binary in the ./bin directory.

**Comment by ldaniels528** [comment]
Fixed. See release v0.19.1

**Comment by mariussoutier** [comment]
Why don't you just release the Play app?

**Comment by ldaniels528** [comment]
Do you mean as a separate project? Because the Play app's binary is available as part of each release (starting with 0.19.0)

**Comment by mariussoutier** [comment]
Ok I just download the 0.19.1 which only contains the Play app as far as I can see and that works.
